webapps/frontend/ui/cockpit/client/scripts/directives/time-to-live/time-to-live-dialog.js

```diff
@@ -26,17 +26,26 @@ const Controller = [
   'Notifications',
   '$translate',
   '$location',
+  'hasPlugin',
   function(
     camAPI,
     $modalInstance,
     $scope,
     Notifications,
     $translate,
-    $location
+    $location,
+    hasPlugin
   ) {
     const resource = camAPI.resource($scope.resource);
+    const hasBatchOperationPlugin = hasPlugin(
+      'cockpit.navigation',
+      'batch_operation'
+    );
     $scope.showLinkToBatchProcess =
-      $scope.$parent?.resource !== 'case-definition';
+      hasBatchOperationPlugin &&
+      ['process-definition', 'decision-definition'].includes(
+        $scope.$parent?.resource
+      );
```

Comment:
❓Why did you change this condition from `$scope.$parent?.resource !== 'case-definition'`?